modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/util/CPMediaImpl.java

```diff
@@ -62,6 +62,20 @@ public CPMediaImpl(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		_title = fileEntry.getTitle();
 	}
 
+	public CPMediaImpl(long groupId) throws PortalException {
+		String defaultUrl = CommerceMediaResolverUtil.getDefaultUrl(groupId);
+
+		_downloadUrl = defaultUrl;
+
+		_id = 0;
+		_url = defaultUrl;
+
+		_thumbnailUrl = defaultUrl;
+
+		_mimeType = null;
+		_title = null;
```

Comment:
@alecsloan I think we can remove the new lines and sort this alphabetically.

Reply:
Hey @dsitu I can do alphabetical but I have to leave spaces after referencing defaultUrl

Reply:
yep that looks good to me, thanks!